src/libfsm/state.c

```diff
@@ -9,6 +9,20 @@
 
 #include "internal.h"
 
+static int
+fsm_state_cmpedges(const void *a, const void *b)
+{
+	const struct fsm_edge *ea, *eb;
+
+	assert(a != NULL);
+	assert(b != NULL);
+
+	ea = a;
+	eb = b;
+
+	return (ea->symbol > eb->symbol) - (ea->symbol < eb->symbol);
```

Comment:
If the ordering matters, better comment about it (I think we rely on it, during iteration)